CHANGELOG.md

```diff
@@ -6,15 +6,19 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+-   Updated ESDTToken fields to match new API response: `token` is now `identifier`
```

Comment:
👍  but can we use the pattern as below? With the description being a link to the PR.